src/Model/Shared/CustomField.php

```diff
@@ -288,37 +310,37 @@ public function setName(?string $name): self
     }
 
     /**
-     * @return bool|null
+     * @return int|null
```

Comment:
Did you intentionally change these back to `int`?

Reply:
Nope, by mistake after copying model from another branch.